maintainers/maintainer-list.nix

```diff
@@ -9275,6 +9275,12 @@
     githubId = 2212422;
     name = "uwap";
   };
+  v = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add veleth`` and maybe change your name to veleth.